frontend/mr-admin-flate/src/components/modal/SlettAvtaleGjennomforingModal.tsx

```diff
@@ -0,0 +1,127 @@
+import { Button, Heading, Modal } from "@navikt/ds-react";
+import {
+  ApiError,
+  Avtale,
+  Opphav,
+  Tiltaksgjennomforing,
+} from "mulighetsrommet-api-client";
+import { useEffect } from "react";
+import classNames from "classnames";
+import { useNavigate } from "react-router-dom";
+import { XMarkOctagonFillIcon } from "@navikt/aksel-icons";
+import styles from "../modal/Modal.module.scss";
+import { Lenkeknapp } from "../lenkeknapp/Lenkeknapp";
+import { UseMutationResult } from "@tanstack/react-query";
+
+interface SlettTiltaksgjennomforingModalprops {
+  modalOpen: boolean;
+  handleCancel: () => void;
+  data: Tiltaksgjennomforing | Avtale;
+  mutation: UseMutationResult<string, unknown, string>;
+  dataType: "tiltaksgjennomforing" | "avtale";
+}
+
+const SlettAvtaleGjennomforingModal = ({
+  modalOpen,
+  handleCancel,
+  data,
+  mutation,
+  dataType,
+}: SlettTiltaksgjennomforingModalprops) => {
+  const navigate = useNavigate();
+
+  const fraArena = data?.opphav === Opphav.ARENA;
+
+  useEffect(() => {
+    if (mutation.isSuccess) {
+      navigate(`/${dataType}er`);
+      return;
+    }
+  }, [mutation]);
+
+  const handleDelete = () => {
+    mutation.mutate(data.id);
+  };
+
+  function headerInnhold() {
+    return (
+      <div className={styles.heading}>
+        <XMarkOctagonFillIcon className={styles.warningicon} />
+
+        {fraArena ? (
+          <span>
+            {dataType === "tiltaksgjennomforing"
+              ? "Gjennomføringen"
+              : "Avtalen"}{" "}
+            kan ikke slettes
+          </span>
+        ) : mutation.isError ? (
+          <span>Kan ikke slette «{data.navn}»</span>
+        ) : (
+          <span>Ønsker du å slette «{data.navn}»?</span>
+        )}
+      </div>
+    );
+  }
+
+  function modalInnhold() {
+    return (
+      <>
+        {fraArena ? (
+          <p>
+            {dataType === "tiltaksgjennomforing"
+              ? "Gjennomføringen"
+              : "Avtalen"}{" "}
+            «{data.navn}» kommer fra Arena og kan ikke slettes her
+          </p>
+        ) : mutation?.isError ? (
+          <p>{(mutation.error as ApiError).body}</p>
+        ) : (
+          <p>Du kan ikke angre denne handlingen</p>
+        )}
+        <div className={styles.knapperad}>
+          {fraArena ? null : mutation?.isError ? (
+            <Lenkeknapp
+              to={`/avtaler/skjema?tiltaksgjennomforingId=${data?.id}`}
```

Comment:
Er dette korrekt rute for å redigere en gjennomføring? Skulle det ikke vært `/tiltaksgjennomforinger/skjema?tiltaksgjennomforingId`?